packages/styles/globals.import.styl

```diff
@@ -85,6 +113,49 @@ textarea.form-control
 .rm-pad-btm
   padding-bottom 0
 
+.rm-margin-top
+  margin-top 0
+
+.rm-margin-btm
+  margin-bottom 0
+
+.inline
+  display inline-block
+  vertical-align middle
+
+.toggleSwitch
```

Comment:
Oh I thought I removed that. It probably will be in the future and I was preparing to use it for the email hiding option.